### PR TITLE
translated more documents

### PR DESCRIPTION
**Pull request by adebisi5**
more documents translated into spanish.

### CODE REVIEW

File: terraform/.terraform.lock.hcl [hunk 1]
```diff
@@ -1,38 +0,0 @@
-# This file is maintained automatically by "terraform init".
```
Comment (Twipped) [inline terraform/.terraform.lock.hcl:1]: Err, why did you delete the infrastructure lock file?

**Review by Twipped — CHANGES_REQUESTED** [review]

**Comment by adebisi5** [comment]
oops a mistake. I'll fix it after hurricane is over.

Sent from Proton Mail mobile

-------- Original Message --------

File: terraform/.terraform.lock.hcl (hunk 1 above)
Comment (adebisi5) [reply terraform/.terraform.lock.hcl]: fixed it.

**Comment by Twipped** [comment]
Thanks!
